DotNetCoreMasters/Repositories/Implementation/ItemRepository.cs

```diff
@@ -9,49 +10,47 @@ namespace Repositories.Implementation
 {
     public class ItemRepository : IItemRepository
     {
-
-        public ItemRepository()
+        private IItemContext _itemContext;
+        public ItemRepository(IItemContext itemContext)
```

Comment:
where did IItemContext come from? It should be DataContext

Reply:
done renaming to DataContext